module/VuFind/src/VuFind/Form/Form.php

```diff
@@ -548,14 +548,24 @@ protected function parseConfig($formId, $config, $params)
         $elements = [];
         $configuredElements = $this->getFormElements($config);
 
-        // Add sender contact name & email fields
+        // Defaults for sender contact name & email fields:
         $senderName = [
-            'name' => 'name', 'type' => 'text', 'label' => 'feedback_name',
-            'group' => '__sender__'
+            'name' => 'name',
+            'type' => 'text',
+            'label' => $this->translate('feedback_name'),
```

Comment:
Was it a bug that this was not translated before, or are we now double-translating it?

Reply:
Previously it was translated in the loop below, but now it resembles the result of changes made in the loop. In other words, previously it was a configuration entry, while now it's an internal field definition. So, neither.

Reply:
Ahh, of course, makes sense!